packages/calcite-components/src/components/tile-select-group/interfaces.ts

```diff
@@ -1 +1,2 @@
 export type TileSelectGroupLayout = "vertical" | "horizontal";
+export type TileSelectGroupDir = "ltr" | "rtl";
```

Comment:
I thought we had a shared `Dir` type, but it doesn't look that way. Can you move this one to the shared interfaces file? We should be able to use it in more places.

Reply:
I moved `Dir` into the shared interfaces file!